src/server/chromium/crNetworkManager.ts

```diff
@@ -207,10 +207,10 @@ export class CRNetworkManager {
       frame = this._page._frameManager.frame(requestPausedEvent.frameId);
 
     // Check if it's main resource request interception (targetId === main frame id).
-    if (!frame && requestPausedEvent && requestWillBeSentEvent.frameId === (this._page._delegate as CRPage)._targetId) {
```

Comment:
@dgozman For the new popup page created as delegate from main page, the event was not getting fired without this change in chromium. 

Not sure if this is the right way, LMK WDYT.

Reply:
@vigneshshanmugam Good catch. I just traced the protocol myself, and in some cases we are getting the network request from the browser before the frame tree arrives from the renderer. I think this is a good fix!

Reply:
Nice, Thanks for double checking 👍